datasets/HMRC-RTS-Small-area/main.py

```diff
@@ -66,19 +70,10 @@ def process_tab(t):
 #Flow has been changed to Flow Direction to differentiate from Migration Flow dimension
 # -
 
-out = Path('out')
-out.mkdir(exist_ok=True)
-table.to_csv(out / 'observations.csv', index = False)
+cubes.add_cube(scraper, table, info['title'])
+scraper.dataset.family = info['families']
+scraper.catalog.dataset[0].family = info['families']
```

Comment:
From looking into the .py files the main runs (T1, T2, T3, T4, T5) and the output files I can see this is a multi-measure dataset; GBP Total &  Count of Businesses therefore will need split out into two cubes and use the cube class in the way to represent this. Here is an example of how it has previously been done for reference https://github.com/GSS-Cogs/family-trade/blob/master/datasets/ONS-Regional-gross-domestic-product-city-regions/main.py#L464 

how the scraper is handled in each separate notebook (T1 - T5) can change scraper = Scraper(seed="info.json")

it'll pick up the url and family information from the info.json automatically and then lines 74 & 75 can be removed.

Reply:
From a linked data perspective, the output is atomic (i.e. each record only has one measure type + value). I know it's a multiple measure, but it's being represented in the correct way. I'm going to have to read more up on this to see how it's worked in your example next week. _I am **done** for this week._

Reply:
Fixed.